packages/trees/backend/sirius-components-trees/src/main/java/org/eclipse/sirius/components/trees/labels/Style.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.trees.labels;
+
+/**
+ * Fragment's style.
+ *
+ * @author mcharfadi
+ */
+public record Style(String font, String backgroundColor, String foregroundColor, String strikeoutColor, String underlineColor, String borderColor, boolean isStrikedout, UnderLineStyle underLineStyle, BorderStyle boderStyle) { }
```

Comment:
- Use a builder pattern to help create those objects
- Use a more meaningful name (see other comment on the matter)